### PR TITLE
Bw config update+change

### PR DESCRIPTION
**Pull request by Prometheus0000**

### CODE REVIEW

File: config/bartworks.cfg [hunk 1]
```diff
@@ -112,8 +493,6 @@ system {
 
     # Enables or Disables the debug log.
     B:"Enable Debug Log"=false
-
-    # Enables or Disables the Experimental Threaded Material Loader.
     B:"Enable Experimental Threaded Material Loader"=false
```
Comment (bartimaeusnek) [inline config/bartworks.cfg:496]: this can be deleted completely

**Comment by Dream-Master** [comment]
@Prometheus0000

File: config/bartworks.cfg (hunk 1 above)
Comment (Prometheus0000) [reply config/bartworks.cfg:496]: Those lines are already deleted, or do you mean the 'B:"Enable Experimental Threaded Material Loader"=false' line?

**Comment by Prometheus0000** [comment]
The RFC (#5878) is linked from the Issue (#6279).

The issue states both the numbers, and what they represent for what got removed. Should I not expect that people read linked issues from PRs? Or do I need to link it in a comment as well as in a commit text?

File: config/bartworks.cfg (hunk 1 above)
Comment (bartimaeusnek) [reply config/bartworks.cfg:496]: the 'B:"Enable Experimental Threaded Material Loader"=false' line, cause thats been removed

**Comment by Dream-Master** [comment]
@bartimaeusnek

**Pull request merged**